webknossos/pyproject.toml

```
@@ -33,7 +33,7 @@ python = ">=3.7.1,<3.10"
 aiohttp = "^3.8.1"
 attrs = "^21.1.0"
 boltons = "~21.0.0"
-cattrs = "22.2.0"
+cattrs = ">=1.7.1,<=22.2.0"
```

I think specifying the lower end is enough:
```suggestion
cattrs = ">=1.7.1"
```

---

Wouldn't that be susceptible to untested new versions and potential breaking changes, though?